### PR TITLE
Add image building and loading part in README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by xuezhaojun**
The image quay.io/open-cluster-management-io/helloworld-addon can't not been download because of auth issue. So users should build it by themselves.

Signed-off-by: xuezhaojun <zxue@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +33,15 @@ Set environment variables.
 export KUBECONFIG=</path/to/hub_cluster/kubeconfig>
 ```
 
-Override the docker image (optional)
+Build the docker image.
```
Comment (qiujian16) [inline README.md]: build the docker image to run the sample addon.

Comment (xuezhaojun) [reply README.md]: Done

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**